### ISSUE TITLE
Is it possible to log in to an account and read emails under Linux?

### ISSUE THREAD

**Issue by szsyzx** [Tolian500/outlook_autologin#1]
Is it possible to log in to an account and read emails under Linux?

**Comment by Tolian500** [comment]
Yep. It creates windows-like browser so you can login to outlook/teams as if you were using  windows machine.

**Comment by szsyzx** [comment]
What I mean is, can you check and modify the contents of the emails in the mailbox after entering the account and password? Your current script now realizes the login。

**Comment by Tolian500** [comment]
It was not a goal so it can't at the moment. But you can use it as starting point and  can create some logic and automation using Selenim Webdriver which is used here.
